vividus-tests/src/main/resources/story/system/mobile_app/MobileAppStepsTests.story

```diff
@@ -14,6 +14,12 @@ Given I start mobile application with capabilities:
 |app |${app-url}|
 
 
+Scenario: Verify step: 'When I reinstall mobile application with bundle identifier `$bundleId`'
+Meta:
+    @requirementId 2073
+When I reinstall mobile application with bundle identifier `${main-app}`
```

Comment:
No validations here?

Reply:
For example, you could open some view, validate it opened, and then reinstall. In the current implementation of the test, if step does nothing at all we won't get any error